src/constants/aois.js

```diff
@@ -20,6 +20,7 @@ export const BASE_LAYERS = [
   { title: 'gadm-0-admin-areas-feature-layer' },
   { title: 'mask-layer' },
   { title: 'specific-regions-tile' },
+  { title: 'admin_areas_feature_layer' },
```

Comment:
I am pretty sure is possible to improve this.

Reply:
But this base layers should be the ones that are activeLayers on the aoi-scene/config.js. I think you can use them from there as they are exported